src/xb-silo-export.c

```diff
@@ -64,7 +64,7 @@ xb_silo_export_node (XbSilo *self, XbSiloExportHelper *helper, XbSiloNode *sn, G
 		helper->off += xb_silo_node_get_size (sn);
 
 		/* recurse deeper */
-		while (xb_silo_get_node(self, helper->off)->is_node) {
+		while (xb_silo_get_node(self, helper->off)->flags & XB_SILO_NODE_FLAG_IS_ELEMENT) {
```

Comment:
Nitpick: You could fix the spacing after `xb_silo_get_node` while modifying this line